wavefront-istio-adapter/values.yaml

```diff
@@ -0,0 +1,25 @@
+adapter:
+  image: vmware/wavefront-adapter-for-istio
+  tag: 0.1.3
+
+credentials:
+# Define either direct or proxy credentials.
+  direct:
```

Comment:
We can remove this, once the `wavefront` chart dependency is added, we can use `wavefront.wavefront.url` and `wavefront.wavefront.token` instead.